amundsen_application/api/search/v0.py

```diff
@@ -57,6 +71,73 @@ def _validate_search_term(*, search_term: str, page_index: int) -> Optional[Resp
     return None
 
 
+@search_blueprint.route('/table_qs', methods=['POST'])
+def search_table_qs() -> Response:
+    # TODO (ttannis): This method is in development and will be cleaned up in a future PR
+    request_json = request.get_json()
+
+    search_term = get_query_param(request_json, 'term', '"term" parameter expected in request data')
+    page_index = get_query_param(request_json, 'pageIndex', '"pageIndex" parameter expected in request data')
+    filters = request_json.get('filters', {})
+
+    filter_payload = {}
+    valid_categories = ['column', 'database', 'schema', 'table', 'tag']
+    for category in valid_categories:
+        values = filters.get(category)
+        if type(values) == str and len(values) > 0:
+            filter_payload[category] = [values, ]
```

Comment:
nit: extra trailing comma?

Reply:
This is a python thing when it comes to transforming a string into an array. If `values = 'ttannis'` then `[values] =['t', 't', 'a', 'n', 'n', 'i', 's']`  but `[values, ] =['ttannis']`